src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -63,13 +59,17 @@
   <ItemGroup Condition="'$(TargetGroup)' == 'uap101aot'">
     <TargetingPackReference Include="System.Private.Interop" />
     <TargetingPackReference Include="System.Private.Corelib" />
-    <TargetingPackReference Include="System.Private.Threading" /> 
+    <TargetingPackReference Include="System.Private.Threading" />
     <TargetingPackReference Include="Windows" />
     <ProjectReference Include="$(SourceDir)\mscorlib.WinRT-Facade\mscorlib.WinRT-Facade.csproj" />
     <TargetingPackReference Include="System.Private.CoreLib.WinRTInterop" />
-    <ProjectReference Include="..\..\System.IO\src\System.IO.csproj">
+    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj">
       <TargetGroup>uap101aot</TargetGroup>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Runtime.Extensions\ref\System.Runtime.Extensions.csproj">
+      <UndefineProperties>%(ProjectReference.UndefineProperties);TargetGroup</UndefineProperties>
```

Comment:
You may also need to undefined OSGroup.  Also since we don't define any UndefineProperties in any ItemDefinitionGroup I believe using the append syntax here does nothing.